runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -716,6 +708,49 @@ public void onResponse(Call call, Response response) throws IOException
 		});
 	}
 
+	/**
+	 * Determines the world type folder, if any, to which the
+	 * screenshot should be saved.
+	 *
+	 * @return the world type folder name as a string,
+	 * or {@code null} if the world type is a normal world
+	 * (Free, Members, PVP, High-Risk PVP)
+	 */
+	private String getWorldFolderType()
+	{
+		final EnumSet<WorldType> worldTypes = client.getWorldType();
+
+		if (worldTypes.contains(WorldType.DEADMAN_TOURNAMENT))
+		{
+			return "deadman-tournament";
+		}
+		else if (worldTypes.contains(WorldType.SEASONAL_DEADMAN))
+		{
+			return "deadman-seasonal";
+		}
+		else if (worldTypes.contains(WorldType.DEADMAN))
+		{
+			return "deadman";
+		}
+		else if (worldTypes.contains(WorldType.TOURNAMENT))
+		{
+			return "tournament";
+		}
+		else if (worldTypes.contains(WorldType.LAST_MAN_STANDING))
+		{
+			return "last-man-standing";
+		}
+		else if (worldTypes.contains(WorldType.SKILL_TOTAL))
+		{
+			return "skill-total";
+		}
+		else if (worldTypes.contains(WorldType.BOUNTY))
+		{
+			return "bounty";
+		}
```

Comment:
Should `SKILL_TOTAL` and `BOUNTY` be their own subdirectories? Are they considered normal worlds in terms of stats?

Same with `LAST_MAN_STANDING`, but I'm not sure how LMS works because if I remember correctly, Jagex does some stat faking, so that block might be needed.